assets/js/components/community/overrides/SubcommunityCreateForm.js

```diff
@@ -0,0 +1,338 @@
+          },
+          metadata: {
+            slug: "",
+          },
+        }}
+        onSubmit={this.onSubmit}
+      >
+        {({ values, isSubmitting, handleSubmit }) => (
+          <Form onSubmit={handleSubmit} className="communities-creation">
+            <Message hidden={error === ""} negative className="flashed">
+              <Grid container centered>
+                <Grid.Column mobile={16} tablet={12} computer={8} textAlign="left">
+                  <strong>{error}</strong>
+                </Grid.Column>
+              </Grid>
+            </Message>
+            <Grid container centered>
+              <Grid.Row>
+                <Grid.Column mobile={16} tablet={12} computer={8} textAlign="center">
+                  <Header as="h1" className="rel-mt-2">
+                    {i18next.t("Subcommunity request")}
+                  </Header>
+                  <Divider />
+                </Grid.Column>
+              </Grid.Row>
+              <Grid.Row textAlign="left">
+                <Grid.Column mobile={16} tablet={12} computer={8}>
+                  <div className="field">
+                    <Form.Field>
+                      {i18next.t("Do you already have an existing community?")}
+                    </Form.Field>
+                    <Form.Group>
+                      <RadioField
+                        label={i18next.t("Yes")}
+                        checked={hasCommunity === true}
+                        value={i18next.t("Yes")}
+                        onChange={() => {
+                          this.setState({ hasCommunity: true });
+                        }}
+                        fieldPath="metadata.hasCommunity"
+                        disabled={_isEmpty(communities)}
+                      />
+                      <RadioField
+                        label={i18next.t("No")}
+                        checked={hasCommunity === false}
+                        value={i18next.t("No")}
+                        onChange={() => {
+                          this.setState({ hasCommunity: false });
+                        }}
+                        fieldPath="metadata.hasCommunity"
+                      />
+                    </Form.Group>
+                  </div>
+                  {hasCommunity && (
+                    <SelectField
+                      label={
+                        <FieldLabel
+                          icon="user"
+                          label={i18next.t("Community")}
+                          id="community-label"
+                          class="block"
+                        />
+                      }
+                      fieldPath="metadata.community"
+                      options={communities}
+                      required
+                      disabled={_isEmpty(communities)}
+                    />
+                  )}
+                  {!hasCommunity && (
+                    <>
+                      <RemoteSelectField
```

Comment:
Since this component is to be used with the suggestion API, we can't really use it for awards right now.

We also want to filter awards by funder (e.g. show only "EC" awards), so we have to use the search API. 

That said, we discussed IRL and agreed to implement a dropdown with search capabilities like [MembersSearchBar](https://github.com/inveniosoftware/invenio-communities/blob/3fc1922d37a43a57da2f34fb19fdd17d3096d2d5/invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/MemberSearchBar.js#L16)